pkg/ingester/pyroscope/ingest_handler.go

```diff
@@ -46,17 +46,17 @@ func NewIngestHandler(l log.Logger, p ingestion.Ingester) http.Handler {
 }
 
 func (h ingestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	tenantID, _ := tenant.ExtractTenantIDFromContext(r.Context())
 	input, err := h.ingestInputFromRequest(r)
 	if err != nil {
-		_ = h.log.Log("msg", "bad request", "err", err)
+		_ = h.log.Log("msg", "bad request", "err", err, "orgID", tenantID)
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
 
-	tenantID, _ := tenant.ExtractTenantIDFromContext(r.Context())
 	err = h.ingester.Ingest(r.Context(), input)
 	if err != nil {
-		_ = h.log.Log("msg", "pyroscope ingest", "err", err, "tenant_id", tenantID)
+		_ = h.log.Log("msg", "pyroscope ingest", "err", err, "orgID", tenantID)
```

Comment:
Would give it an error level

```suggestion
		_ = level.Error(h.log).Log("msg", "pyroscope ingest", "err", err, "orgID", tenantID)
```

Reply:
the thing is that the h.log is already wrapped with level.Error in the constructor function :shrug:

Reply:
Then forget about it 🙂